.travis.yml

```diff
@@ -90,6 +102,7 @@ install:
     fi
 
 before_script:
+  - git config --global user.name "Travis bot"
```

Comment:
It picks one from the environment referring to the travis build that is good enough for me.